plugins/blip/basics/components/CalendarContainer.js

```diff
@@ -28,6 +28,9 @@ var BasicsUtils = require('./BasicsUtils');
 var ADay = require('./day/ADay');
```

Comment:
Confirmed that I'm not seeing this on my side, but I'm not sure how you could be missing that commit. 🤷‍♂️